site/content/docs/5.2/forms/checks-radios.md

```diff
@@ -445,6 +439,8 @@ Make star ratings look inactive inside or outside a form by adding the `disabled
 
 {{< example >}}
 <fieldset class="star-rating" disabled>
+  <legend class="visually-hidden">Rated 3 stars out of 5</legend>
```

Comment:
Need a `<legend>` inside each `<fieldset>` ?

Reply:
According to [https://www.w3.org/WAI/WCAG21/Techniques/html/H71](https://www.w3.org/WAI/WCAG21/Techniques/html/H71): 'The first element inside the `fieldset` must be a `legend` element, which provides a label or description for the group'.

Reply:
The remaining question is: it's mandatory but should we do it?

For instance, the `.form-control`: https://boosted.orange.com/docs/5.2/forms/form-control/

Some of the examples in the docs don't have a label even if it's mandatory.
Because we think that user will be smart enough to know it and to do it.

Reply:
Since it's a Boosted modification, I guess that's better and easy to implement.